src/rust/dbv2/src/schema.rs

```diff
@@ -149,11 +147,39 @@ diesel::table! {
     }
 }
 
+diesel::table! {
+    market_account_handles (user) {
+        #[max_length = 70]
+        user -> Varchar,
+        #[max_length = 70]
+        handle -> Varchar,
+    }
+}
+
+diesel::table! {
+    balance_updates_by_handle (txn_version, handle, market_id, custodian_id) {
```

Comment:
I think the minimal primary key here is just `handle` right? If `txn_version` is part of the primary key we'll be recording every balance update that ever happens. Not necessary to include `market_id` and `custodian_id` since every handle will only have one distinct tuple of those.